tests/test_Summary.cpp

```diff
@@ -564,6 +572,7 @@ BOOST_AUTO_TEST_CASE(well_keywords) {
     BOOST_CHECK_CLOSE( 2 * 30.2, ecl_sum_get_well_var( resp, 2, "W_3", "WGIT" ), 1e-5 );
     BOOST_CHECK_CLOSE( 2 * 30.3, ecl_sum_get_well_var( resp, 2, "W_3", "WNIT" ), 1e-5 );
     BOOST_CHECK_CLOSE( 2 * 30.16, ecl_sum_get_well_var( resp, 2, "W_3", "WCIT" ), 1e-5 );
+    BOOST_CHECK_CLOSE( 2 * 30.17, ecl_sum_get_well_var( resp, 2, "W_3", "WSIT" ), 1e-5 );
     BOOST_CHECK_CLOSE( 2* (30.6 + 30.7 + 30.8),
```

Comment:
Currently, this (and similar tests with *S*T) fails. It returns 30.17 and not 2 * 30.17. @bska What am I missing?

Reply:
> It returns 30.17 and not 2 * 30.17

It's subtle and not at all your fault&mdash;the code here is correct as far as I can tell.

The problem happens because [`SummaryState::update_well_var()`](https://github.com/OPM/opm-common/blob/89cbb9ae24fb37b88b4c6c84636a4719dcce7d22/src/opm/parser/eclipse/EclipseState/Schedule/SummaryState.cpp#L141) does not recognise `WSIT` as a cumulative total quantity.  Consequently `update_well_var()` just [overwrites](https://github.com/OPM/opm-common/blob/89cbb9ae24fb37b88b4c6c84636a4719dcce7d22/src/opm/parser/eclipse/EclipseState/Schedule/SummaryState.cpp#L147-L148) the value instead of [adding](https://github.com/OPM/opm-common/blob/89cbb9ae24fb37b88b4c6c84636a4719dcce7d22/src/opm/parser/eclipse/EclipseState/Schedule/SummaryState.cpp#L144-L145) the contribution to the previous value.

Looking at the definition of [`is_total()`](https://github.com/OPM/opm-common/blob/89cbb9ae24fb37b88b4c6c84636a4719dcce7d22/src/opm/parser/eclipse/EclipseState/Schedule/SummaryState.cpp#L32-L53) I don't see `*SIT` among the known vectors, so `is_total("WSIT")` will return `false` and we get the overwriting behaviour.  Fixing this **should** be a matter of adding `SIT` to the list of known cumulative quantities.

Reply:
Thanks. I assumed something like this. I will add *SIT.